package.json

```diff
@@ -37,57 +37,58 @@
   "dependencies": {
     "@juggle/resize-observer": "3.3.1",
     "debug": "4.3.2",
-    "media-stream-library": "10.0.2",
-    "react-hooks-shareable": "1.27.0"
+    "react-hooks-shareable": "1.33.0"
   },
   "peerDependencies": {
     "luxon": "^1.25.0",
+    "media-stream-library": "^10.0.2",
```

Comment:
Why?

And maybe add something about it in the Readme.

Reply:
It makes it easier to combine with direct use of media-stream-library without having to update the media-stream-player all the time in sync. It does not affect users of the bundle though, only when you import and run e.g. webpack yourself, at which point the only difference is that you have to install the dependency. I'll update the README.md.